packages/extensionmanager/src/model.ts

```diff
@@ -176,6 +176,16 @@ const EXTENSION_API_PATH = 'lab/api/extensions';
  */
 export type Action = 'install' | 'uninstall' | 'enable' | 'disable';
 
+/**
+ * Additional options for an action.
+ */
+export interface IActionOptions {
+  /**
+   * If true, install the latest version of the entry.
+   */
+  useLatestVersion?: boolean;
```

Comment:
As the backend support a specific version, let's add that as an option:

```suggestion
  /**
   * Install the version of the entry.
   *
   * #### Note
   * This is ignored by all actions except install.
   */
  useVersion?: string;
```